### DISCUSSION TITLE
How to group duplicated dependency in multiple package.json in the same repository into one PR

### DISCUSSION THREAD

**Discussion by ik-hzn** [renovatebot/renovate#20035] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

A configuration the gives one PR for a duplicated dependency found in multiple package.json in the same repository

### What you tried so far.

No idea where to start

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
This should be the default behavior. For the scenario where you have 2+ PRs for the same dependency, what are the branch names?

**Reply by ik-hzn** [reply]
I have these 2 branch names for the `react-i18next` dependency:
- renovate/services/co-worker-ui/react-i18next-12.x
- renovate/services/self-serve-ui/react-i18next-12.x

So we have a package.json in `./services/co-worker-ui` and one in `./services/self-serve-ui`.

**Reply by rarkins** [reply]
It sounds like you or someone in your company has intentionally split your branches by service, probably by setting `additionalBranchPrefix` to ``{{packageFileDir}}/`. In other words, the thing you want to change is caused by your own config.

**Reply by ik-hzn** [reply]
Yes we have this `"additionalBranchPrefix": "{{baseDir}}/",`
Though the guy who did this change isn't working on this project anymore.
What are the implications of removing this? And what would be the reason to have it?

**Reply by rarkins** [reply]
The reason to have it is to split PRs. Ie someone wanted the opposite to what you now want. 

Removing it will solve your problem by auto closing existing PRs and using a common branch name for same dependencies for future PRs

**Reply by ik-hzn** [reply]
Really helpful, thank you!
